features/polyExplorer/src/components/entityList/entityList.jsx

```diff
@@ -5,46 +5,49 @@ import EntityShortInfo from "../entityShortInfo/entityShortInfo.jsx";
 
 import "./entityList.css";
 
-function getStartGroups(entities) {
-    let numberGroups = 0;
-    let numberValues = 0;
-    const keys = Object.keys(entities);
-    for (let e of keys) {
-        numberGroups++;
-        numberValues += entities[e].length;
-        if (numberValues > 15) return keys[numberGroups];
+function determineInitialGroups(entities) {
+    const groups = Object.keys(entities);
+    let remainingValues = 15;
+    for (let i = 0; i < groups.length; i++) {
+        remainingValues -= entities[groups[i]].length;
+        if (remainingValues < 0) return Math.min(i + 2, groups.length);
     }
-    return keys.pop();
+    return groups.length;
 }
 
 function EntityList({ entities, sideLabel }) {
-    const allKeys = Object.keys(entities);
+    const allGroups = Object.keys(entities);
     const [loadedEntities, setLoadedEntities] = useState({});
-    const [toLoadKeys, setToLoadKeys] = useState(allKeys);
+    const [groupsToLoad, setGroupsToLoad] = useState(allGroups);
     const [hasMore, setHasMore] = useState(true);
     const listRef = useRef();
 
     function loadEntities() {
+        // This logic is currently not very efficient for an initial group with
```

Comment:
Instead of inline comments, why not a docstring? It would be the only one, I know, but seeing "currently" in a comment makes me wonder what it refers to.

Reply:
To this function - `loadEntities` is supposed to only load a few more entities than fit on the screen, but the current logic loads tons more because it's based on (entire) groups, not (individual) entities. A doc string would make sense if we want to expose this information to the _caller_, so it could be one at the component level (since `loadEntities` is private). But given how we don't use any doc strings, doesn't seem like the time to start. Like this, it's as close to the code it refers to as possible. If I was to move this to _polyLook_, I'd most definitely document this though.

Reply:
At least try to avoid ambiguity by not using "this" and "currently". Possibly replicate the code so that when it changes, we know that the comment is out of sync.

Reply:
Alrighty, done!